pkg/controller/installation/validation_test.go

```diff
@@ -791,6 +791,31 @@ var _ = Describe("Installation validation tests", func() {
 		})
 	})
 
+	Describe("validate CalicoNetwork PolicySetupTimeoutSeconds", func() {
+		It("should return an error when PolicySetupTimeoutSeconds is negative", func() {
+			negative := int32(-1)
+			instance.Spec.CalicoNetwork.PolicySetupTimeoutSeconds = &negative
+			err := validateCustomResource(instance)
+			Expect(err).To(MatchError("Installation CNI spec.PolicySetupTimeoutSeconds negative value is not valid"))
+		})
+		It("should return an error when PolicySetupTimeoutSeconds is set for a non-IPTables dataplane", func() {
```

Comment:
nit, but it's nice to have a newline between `It()` clauses for clarity.

Reply:
Fixed